quantum_systems/time_evolution_operators/operator.py

```diff
@@ -192,7 +192,6 @@ def is_two_body_operator(self):
 
     def u_t(self, current_time):
         np = self._system.np
-
```

Comment:
Could you undo this extra line to avoid unnecessary changes in files that are not changed?

Reply:
It looks fine on my end :S

Reply:
Ah, I see I've just written the complete opposite of what I meant 😅 Could you re-insert the extra line? That is, do not include a change in this file as the only change is the removal of a line.